libcipher/caesar.py

```diff
@@ -0,0 +1,76 @@
+__author__ = 'kskrishnasangeeth'
+
+import string
+
+
+def cipher_helper(message, key):
```

Comment:
Since `cipher_helper` is a _private_ method, it is practice to start them with a double underscore (TIP: pronouced **dunderr** or **under under**). Same case with `check_exceptions` as well. 

For more info check out: http://stackoverflow.com/a/1301369/235055